app/views/fileChecksFailed.scala.html

```diff
@@ -1,5 +1,5 @@
 @import views.html.partials.progressIndicator
-@(name: String, isJudgmentUser: Boolean)(implicit request: RequestHeader, messages: Messages)
+@(name: String, isJudgmentUser: Boolean, fileStatus: String = "")(implicit request: RequestHeader, messages: Messages)
```

Comment:
Think the `fileStatus` value should be list. See comments above, given there is the possibility that could have zip and password protected errors.